### PR TITLE
SLM-256 Accessibility audit high priority issues 2

### PR DESCRIPTION
**Pull request by mikehalmamoj**

### CODE REVIEW

File: server/routes/barcode/image/GenerateBarcodeImageController.test.ts [hunk 1]
```diff
@@ -70,6 +70,12 @@ describe('GenerateBarcodeImageController', () => {
               barcodeImageUrl: 'barcode-address-image-url',
               barcodeImageName: `SendLegalMail-John-Smith-${moment().format('YYYY-MM-DD')}.png`,
               recipientName: 'John Smith',
+              copyButtonHtml: `Copy <span class="govuk-visually-hidden"> barcode image for John Smith (copies PNG file SendLegalMail-John-Smith-${moment().format(
+                'YYYY-MM-DD'
+              )}.png of approximate size 0.2 Megabytes to the Clipboard)</span>`,
+              downloadButtonHtml: `Download <span class="govuk-visually-hidden"> barcode image for John Smith (downloads PNG file SendLegalMail-John-Smith-${moment().format(
+                'YYYY-MM-DD'
+              )}.png of approximate size 0.2 Megabytes)</span>`,
```
Comment (mikehalmamoj) [inline server/routes/barcode/image/GenerateBarcodeImageController.test.ts:78]: All file downloads need to specify the file type and size for users of screen readers.

File: server/views/pages/barcode/choose-barcode-option.njk [hunk 2]
```diff
@@ -19,35 +19,37 @@
...
+      <div class="govuk-grid-column-two-thirds">
+        <form action="/barcode/choose-barcode-option" method="post" novalidate id="choose-barcode-option-form">
+          <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
+
+            {{ govukRadios({
+              fieldset: {
+                legend: {
+                  text: 'How do you want to print the barcodes?',
+                  isPageHeading: true,
+                  classes: 'govuk-fieldset__legend--xl'
```
Comment (mikehalmamoj) [inline server/views/pages/barcode/choose-barcode-option.njk:32]: For pages with a single question use the govuk components to set page headings as these are built with accessibility in mind - we created our own headings and created visually hidden labels that didn't match visual labels thus breaking some screen reader functionality linking labels to inputs.

File: server/views/pages/barcode/edit-contact-details.njk [hunk 3]
```diff
@@ -30,7 +30,7 @@
           <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
           <input type="hidden" id="contactId" name="contactId" value="{{ form.contactId }}" />
 
-          {% set errorDob = errors | findError('dob') %}
+          {% set errorDob = errors | findError('dob-day') %}
```
Comment (mikehalmamoj) [inline server/views/pages/barcode/edit-contact-details.njk:33]: Fix from last PR - errors weren't highlighted on DoB.

**Review by yasinmustafa — APPROVED** [review]
LGTM

**Pull request merged**